rustls-platform-verifier/src/verification/others.rs

```diff
@@ -154,7 +160,7 @@ impl Verifier {
 
         #[cfg(target_arch = "wasm32")]
         {
-            root_store.add_trust_anchors(webpki_roots::TLS_SERVER_ROOTS.iter().map(|root| {
+            root_store.add_trust_anchors(webpki_roots::TLS_SERVER_ROOT_CERTS.iter().map(|root| {
```

Comment:
This looks wrong? If this needs to be `TLS_SERVER_ROOT_CERTS`, it should come from the `webpki_root_certs` crate, right? Are we not compiling for `wasm32` in CI?

Reply:
[wasm isn't checked on the CI](https://github.com/rustls/rustls-platform-verifier/blob/8080972e5fb8f5b5aeb0172abac18d5834d11ced/.github/workflows/ci.yml#L70-L75) and seems broken on main, at least on my computer. The function `add_trust_anchors` is not even in `rustls` anymore. 
Maybe we should fill a separate issue for wasm? I've changed to `webpki_root_certs` as you rightfully pointed out, but this won't be sufficient to fix the whole target.